interface/src/app/utils.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test for hypergeometric test function in utils.js', function() {
+  beforeEach(module('adage.utils'));
+
+  var mockService;
+
+  beforeEach(function() {
+    angular.mock.inject(function($injector) {
```

Comment:
Just so you know, this line can also be written as:
```
  inject(function($injector) {
    // ...
  });
```
because angular.mock.inject() is introduced to the global scope (window) too;

Or the whole **beforeEach** can be more compact like this:
```
beforeEach(inject(function(MathFuncts) {
  mockService = MathFuncts;
}));
```
(Just my two cents on unit tests. After spending some serious time reading Angular doc, I found Angular's injector and injectable types tricky to understand.)